frontend/src/__tests__/integration/pages/modelServing/ServingRuntimeList.stories.tsx

```diff
@@ -102,6 +149,6 @@ export const DeployModel = {
     const canvas = within(canvasElement);
     await canvas.findByText('ovms', undefined, { timeout: 5000 });
 
-    await userEvent.click(canvas.getByText('Deploy model', { selector: 'button' }));
+    await userEvent.click(canvas.getAllByText('Deploy model', { selector: 'button' })[0]);
```

Comment:
This is an indication that we really should be adding test ids -- this is not fun for QE to write automation either.